pkgs/development/python-modules/resolvelib/default.nix

```diff
@@ -12,13 +13,16 @@ buildPythonPackage rec {
   # is compatible with Ansible
   # https://github.com/NixOS/nixpkgs/pull/128636
   # https://github.com/ansible/ansible/blob/devel/requirements.txt
-  version = "0.5.5";
+  version = "0.8.1";
```

Comment:
Did you read the comment above?

Reply:
Yes, I did. Is the reason why it's marked as DRAFT.

Reply:
https://github.com/NixOS/nixpkgs/pull/133064#issuecomment-894791931

I'm not sure how to tackle that. I might as well drop maintainership for ansible, because I'm using it from pip these days.